edenia/proton-affiliate
# Fix/resolve conflicts

### Resolve conflicts

### What does this PR do?

Resolve conflicts to perform a production release

### Steps to test

1. Run the project locally
2. Check the UI changes.
3. Test the contract.
4. Check the results.
